css-writing-modes-3/text-combine-upright/value-all-and-inherit-001.htm

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSS Writing Modes Test: text-combine-upright :all</title>
+    <link rel="author" title="Shinsuke Matsuki" href="mailto:shinsuke.matsuki@access-company.com">
+    <link rel="help" href="http://www.w3.org/TR/css-writing-modes-3/#text-combine-upright">
+    <link rel="match" href="reference/value-all-and-inherit-001-ref.htm">
+    <meta name="assert" content="Two white triangles does not pointing to the right. Pointing up.">
```

Comment:
This assertion should say what the spec assumes. In this case the spec defines the property inherits and the property applies to non-replaceable inline boxes.